Software/Raspberry_Pi/pi/base_detect.py

```diff
@@ -0,0 +1,124 @@
+"""Main script to run the object detection routine."""
+import argparse
+import sys
+import time
+import cv2
+from tflite_support.task import core
+from tflite_support.task import processor
+from tflite_support.task import vision
```

Comment:
Are we still using MobileNet/EfficientNet?  Didn't we need to measure pure-OpenCV solutions for lag reduction?

Reply:
I was using efficientnet before which is SoTA. Mobilenet is an older model that has less parameters and computations than yolo and efficientnet but had good bounding box detection with better latency, so I decided to go with it